### PR TITLE
Add Featured Artists article

### LINKED ISSUE #2203: Create a Featured Artists article
should include:
- general description
- artists list
- short descriptions for artists
- official songs

### PR DESCRIPTION
**Pull request by cl8n**
closes #2203

### CODE REVIEW

File: wiki/Featured_Artists/en.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Featured Artists
+
+<!-- TODO: include some history about featured artists / mappers' guild, maybe in /History_of_osu! too -->
+
+**Featured artists** are music artists that have worked with osu! to make a selection of their tracks available for unrestricted use in the community. Some of the featured artists have also composed tracks specifically for osu! and released them as osu! exclusives.
```
Comment (TPGPL) [inline wiki/Featured_Artists/en.md]: unrestricted use -> fair use?

Comment (cl8n) [reply wiki/Featured_Artists/en.md]: i dont think that's what fair use means (if you're talking about copyright-related stuff)

Comment (Joehuu) [inline wiki/Featured_Artists/en.md]:
```suggestion
**Featured artists** are music artists that have made an agreement with osu! to make a selection of their tracks available for unrestricted use in the community. Some of the featured artists have also composed tracks specifically for osu! and released them as osu! exclusives.
```

Comment (cl8n) [reply wiki/Featured_Artists/en.md]: is the difference important? i think "worked with" sounds more natural

Comment (cl8n) [reply wiki/Featured_Artists/en.md]: yeah "fair use" implies that you're working around a copyright, but in this case osu! has license to use the music so they don't have to worry about fair use

**Pull request merged**